src/ducks/future/selectors.js

```diff
@@ -0,0 +1,52 @@
+import { createSelector } from 'reselect'
+import addDays from 'date-fns/add_days'
+
+import { getRecurrences } from 'selectors'
+import { getFilteredAccountIds } from 'ducks/filters'
+import { nextDate, STATUS_FINISHED } from 'ducks/recurrence'
+
+export const getMaxDate = () => {
+  return addDays(new Date(Date.now()), 30)
```

Comment:
At the moment, the text is "balance in 30 days". This is a good question though. @fffflo ?

Reply:
the 2 options were : end of month forecast vs 30 day forecast. We believe that more than 30 days forecast is unrelevant because  level of uncertainty increases over time.
the drawback of end of month is that you have poor results if your income is changing beetwen arriving end of month or beginning of next month.  and you don't have a lot of insights when you are late in the month.
so we prefer 30 day forecast, with a focus on the lowest point the user will face during the period. 
if your point was to write down the actual date vs "in 30 days" : that's open to improvement or later changes with more user research and feedbacks

Reply:
My point was about the end of month forecast. And the exemple with the income coming at the end of the month or at the beginning of the next one gives me enough information to understand the choice. Thanks!